src/Compiler/Service/ServiceLexing.fsi

```diff
@@ -344,6 +344,9 @@ module FSharpKeywords =
     /// Keywords paired with their descriptions. Used in completion and quick info.
     val KeywordsWithDescription: (string * string) list
 
+    /// A lookup from keywords to their descriptions
+    val KeywordsDescriptionLookup: System.Collections.Generic.IDictionary<string, string>
```

Comment:
Could this be a `string -> string option` method please, `GetKeywordDescription`?  Rather than revealing an IDictionary?